opta/commands/apply.py

```diff
@@ -334,6 +326,39 @@ def _apply(
         )
 
 
+def _verify_semver(old_semver_string: str, current_semver_string: str) -> None:
+    if old_semver_string not in [DEV_VERSION, ""] and current_semver_string not in [
+        DEV_VERSION,
+        "",
+    ]:
+        old_semver = semver.VersionInfo.parse(old_semver_string)
+        current_semver = semver.VersionInfo.parse(current_semver_string)
+        if old_semver > current_semver:
+            raise UserErrors(
+                f"You're trying to run an older version of opta (last run with version {old_semver}). "
+                "Please update to the latest version and try again!"
+            )
+
+        current_upgrade_warnings = sorted(
+            [
+                (k, v)
+                for k, v in UPGRADE_WARNINGS.items()
+                if current_semver.compare(k) >= 0 and old_semver.compare(k) < 0
```

Comment:
```suggestion
                if current_semver >= k and old_semver < k
```

Strings will be implicitly converted to `VersionInfo` objects by the comparison operators.

Reply:
done